### DISCUSSION TITLE
Linux version maybe one day?

### DISCUSSION THREAD

**Discussion by mxmilkiib** [bitgapp/eqMac#864] [Ideas]
This is a neat looking app! But i only ever use Linux. +1 for a possible Linux version some year, possibly maybe! Thanks for your time.

**Comment by macuserguru** [comment]
I'm interested too!

**Comment by fastzombies** [comment]
I use Easy Effects on Ubuntu.  Lots of plugins, seems like it can do everything eqMac can do.  I install from flatpak.

**Comment by ragauskl** [comment]
Thanks for sharing an alternative, there have been discussions about making eqMac cross platform but taking everything into account it may still be a long while before cross platform support is implemented

**Comment by fastzombies** [comment]
Competing with FOSS and making eqMac cross platform is going to be a steep hill to climb.  I may use it since I think we get 3 devices, but Easy Effects is really good and the plugin architecture makes it a community project with lots of options.  Kinda hard to beat.

I see a lot of eqMac issues coming in, I would put resources into making eqMac flawless and the only app anyone thinks of for MacOS audio before expanding my market.

**Reply by ragauskl** [reply]
That is part of what I meant by 'taking everything into account it may be a while', current focus is on making current state as stable as it can be, but there are more factors at play as well.

If talking about competing with alternatives FOSS or not, I think there would easily be audience for eqMac (even if not as much as on macOS) it would still have more features (taking into account everything planned in nearest future) including very likely integration with Cursr which I'm working on. The integration would allow for audio sharing and control between devices, so while still a hill to climb may not be as steep as it may seem

**Reply by fastzombies** [reply]
I look forward to trying it on linux as eqMac is a more polished and nicer looking app.  I hope you find enough audience to make it worthwhile.  People like myself with both Mac and Linux (Mac is work and Ubuntu is home machines) I would certainly be interested in one audio app for both systems.

Dumping a settings or backup file that can be restored would be nice to have as well so the EQ settings don't have to be manually duplicated.
